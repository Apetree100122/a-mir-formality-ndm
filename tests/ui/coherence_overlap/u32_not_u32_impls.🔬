// Test that a positive and negative impl for the same type (`u32`, here) is rejected.
[
    crate core {
        trait Foo<> where [] {}
        impl<> Foo<> for u32 where [] {}
        impl<> !Foo<> for u32 where [] {}
    }
]
