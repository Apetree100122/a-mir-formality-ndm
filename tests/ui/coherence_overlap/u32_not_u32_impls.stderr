Error: check_trait_impl(impl <> Foo < > for (rigid (scalar u32)) where [] { })

Caused by:
    failed to disprove {! Foo((rigid (scalar u32)))} given {}, got {Constraints { env: Env { variables: [], coherence_mode: false }, known_true: true, substitution: {} }}
